internal/host/monitor_test.go

```diff
@@ -149,6 +151,8 @@ var _ = Describe("TestHostMonitoring", func() {
 			{Status: api.Success, ValidationId: string(models.HostValidationIDOcsRequirementsSatisfied)},
 			{Status: api.Success, ValidationId: string(models.HostValidationIDLsoRequirementsSatisfied)},
 		}, nil)
+		mockOperators.EXPECT().GetCPURequirementForRole(gomock.Any(), gomock.Any()).AnyTimes()
```

Comment:
Return missing.

Reply:
as I mentioned before, it returns the default values for the types: `0` and `nil`